airflow/logging_backends/base_logging_backend.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class BaseLoggingBackend(object):
+    def __init__(self, **kwargs):
+        self.schema_name = kwargs.get('schema_name')
+
+    def get_logs(self,**kwargs):
+        """
```

Comment:
Nit is that for a one line comment you can put the triple quotes on the same line